me.hergert.Schemes.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id" : "me.hergert.Schemes",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "schemes",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gir-1.0",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "libpanel",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Ddocs=disabled",
+                "-Dintrospection=disabled",
+                "-Dvapi=false"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://download.gnome.org/sources/libpanel/1.1/libpanel-1.1.0.tar.xz",
+                    "sha256" : "a19f6ecdbc82e601be38da0a93ebd913f4c43769fe4ab4f334c35deb8b3a2f11"
+                }
+            ]
+        },
+        {
+            "name" : "schemes",
+            "buildsystem" : "meson",
+            "sources" : [
```

Comment:
It would be nice if you configured [external-data-checker](https://github.com/flathub/flatpak-external-data-checker) for these sources.

Here is an exmaple: https://github.com/flathub/com.spotify.Client/blob/9d99402ddf468f381673b79dcf4fe1803569897b/com.spotify.Client.json#L55-L62